src/Models/Scopes/Product/WithProductAttributesScope.php

```diff
@@ -26,7 +26,8 @@ public function apply(Builder $builder, Model $model)
             ->addSelect($builder->getQuery()->from . '.entity_id AS id')
             ->addSelect($builder->getQuery()->from . '.sku')
             ->addSelect($builder->getQuery()->from . '.visibility')
-            ->addSelect($builder->getQuery()->from . '.type_id AS type');
+            ->addSelect($builder->getQuery()->from . '.type_id AS type')
+            ->addSelect($builder->getQuery()->from . '.created_at');
```

Comment:
```suggestion
            ->addSelect($model->getQualifiedCreatedAtColumn());
```
https://github.com/laravel/framework/blob/9c44052743b0ee7f3adea36994918b4d8019e8b3/src/Illuminate/Database/Eloquent/Concerns/HasTimestamps.php#L162